core/bump_objects.ts

```diff
@@ -93,12 +93,15 @@ export function bumpIntoBoundsHandler(
   workspace: WorkspaceSvg,
 ): (p1: Abstract) => void {
   return (e) => {
+    console.log('got event');
```

Comment:
Clean up the logging in this file?

Reply:
Ah my bad, forgot to clean these up when I rebased.